test/testdata/lsp/completion/untyped_receiver.rb

```diff
@@ -4,5 +4,6 @@
 sig {params(x: T.untyped).void}
 def foo(x)
   x.
-  # ^ completion: (nothing)
+  # ^ completion: (call site is T.untyped)
+  # ^ apply-completion: [A] item: 0
```

Comment:
ooc, why is the rbedited empty? shouldn't it have identical contents to the input?

Reply:
Uh.  Apparently `update_testdata_exp.sh` is not updating the `.rbedited` files.  Let me fix this.

Reply:
ah yeah that's intentional, the testdata exp script is _only_ for `.*exp` files
it's designed that way so it doesn't have to build and test LSP.
what do you think about making a separate script, and then potentially a wrapper script that calls both? (we already have `update_exp_files.sh` which does _everything_, which is kind of slow because it builds ruby and runs the srb init tests...)

Reply:
`update_exp_files.sh` doesn't update the necessary files either.